src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import ToDo from '../../types/ToDo';
+import { UserInfo } from '../UserInfo';
+import cn from 'classnames';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  todo: ToDo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, id, userId } = todo;
+
+  const foundedUser = usersFromServer.find(
+    checkedUser => checkedUser.id === userId,
+  );
```

Comment:
The variable name `foundedUser` is not semantically correct. It should be `foundUser` to accurately describe the action of finding a user.